tools/wpt/run.py

```diff
@@ -354,6 +354,11 @@ def setup_kwargs(self, kwargs):
                 kwargs["webdriver_binary"] = webdriver_binary
             else:
                 raise WptrunError("Unable to locate or install msedgedriver binary")
+        if kwargs["browser_channel"] == "dev":
+            logger.info("Automatically turning on experimental features for Edge Dev")
+            kwargs["binary_args"].append("--enable-experimental-web-platform-features")
+            # HACK(Hexcles): work around https://github.com/web-platform-tests/wpt/issues/16448
```

Comment:
You probably don't need this. It's a ChromeDriver-specific issue related to its releasing process.

Reply:
Hmm, I saw this line in the logs: `This version of ChromeDriver has not been tested with Chrome version 76.` so 1. it looks like there are more string that need to be replaced in MSEdgeDriver 2. you might also need to disable build check before you can release MSEdgeDriver along with Edge in various channels.

Reply:
Removed the workaround. We'll update the ChromeDriver message string in future releases.